README.md

```diff
@@ -1,3 +1,7 @@
+# Testcontainers
```

Comment:
Should we be explicit here in the title?
```suggestion
# Testcontainers for Go
```

Reply:
Mmmm I now see 🤔 all the langs has Testcontainers as H1 in their main README, so it's fine to have it here.

Please not consider this suggestion 🙏